src/components/common/YourCompany/FolioInformation.vue

```diff
@@ -46,7 +46,7 @@ export default class FolioInformation extends Mixins(CommonMixin) {
     if (this.isCorrectionFiling && this.getCorrectedFiling?.header) {
       return this.getCorrectedFiling.header.folioNumber
     } else if (this.isAlterationFiling) {
-      return this.getSnapshotFolioNumber
```

Comment:
I deleted this getter since this was the only place it was used, and instead used the code on the right.